vcpkg.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
   "name": "ear-production-suite",
-  "version": "0.6.0.0",
+  "version-string": "0.6.0.0",
```

Comment:
Need to be careful that we remember to update this. Not sure if there's a way to automate it using something along the lines of the versioning stuff we implemented earlier?